### PR TITLE
Added MbStrFunctionsFixer (master)

### PR DESCRIPTION
**Pull request by Slamdunk**
`master` PR for https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2082

Instead of `1.11` manual search, here we use `AbstractFunctionReferenceFixer` help.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -336,6 +336,11 @@ Choose from the list of available fixers:
                          PHP keywords MUST be in lower
                          case.
 
+* **mb_str_functions**
```
Comment (GrahamCampbell) [inline README.rst]: mb_string_functions would be a better name perhaps?

**Comment by Slamdunk** [comment]
fabbot.io proposes
```diff
-namespace PhpCsFixer\Tests\Fixer\Alias;
+namespace PhpCsFixer\tests\Fixer\Alias;
```
but it should be an error, since all other classes too have uppercase firt letter for "Tests"

**Comment by GrahamCampbell** [comment]
Yeh, just ignore fabbot.io.

File: README.rst (hunk 1 above)
Comment (Slamdunk) [inline README.rst]: The name must be the same of https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2082 where I didn't receive feedback for that.
I chose that name for the similarity with real functions name.

I kindly ask the opinion of another maintener before changing it in two separate PRs

**Comment by keradus** [comment]
Thank you @Slamdunk

**Pull request closed without merging**